config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -814,13 +814,12 @@ periodics:
       - --extract=ci/fast/latest-fast
       - --extract-ci-bucket=k8s-release-dev
       - --gcp-master-image=gci
-      - --gcp-node-image=gci
```

Comment:
Removing this line looks good to me because the default on GKE is COS (formerly known as GCI = Google Cloud Image)

Reply:
I actually intended to remove `--gcp-master-image=gci` argument as the release jobs defined at https://github.com/kubernetes/test-infra/blob/master/releng/test_config.yaml#L243 has only `--gcp-node-image` argument passed. (mistakenly removed `--gcp-node-image` flag)

I shall change this PR to retain `--gcp-node-image` and remove `gcp-master-image` for `ci-kubernetes-e2e-gci-gce` job.